LocalSettings.php

```diff
@@ -2096,13 +2096,16 @@
 	),
 	'wgManageWikiPermissionsBlacklistRights' => array(
 		'default' => array(
+			// arrays
```

Comment:
why is it necessary to add a comment? there's no comment on the other probably hundred of arrays in this file

Reply:
I've removed it now.